auth/src/main/java/org/aerogear/auth/utils/AuthStateManager.java

```diff
@@ -0,0 +1,60 @@
+package org.aerogear.auth.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import org.aerogear.auth.credentials.OIDCCredentials;
+import org.json.JSONException;
+
+/**
+ * Saves, retrieves and delete a token.
+ */
+public class AuthStateManager {
+
+    private static final String STORE_NAME = "org.aerogear.android.auth.AuthState";
+    private static final String KEY_STATE = "state";
+
+    private SharedPreferences mPrefs;
```

Comment:
This is initialised only in constructors. Should be final.
Do we use the 'm' prefix for member variables?